### PR TITLE
Redmonk interview blog post

### PR DESCRIPTION
**Pull request by nimbinatus**
Still need links, images, CTA UTM link

### CODE REVIEW

File: themes/default/content/blog/redmonk-pulumi-developer-first-infra/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Pulumi and RedMonk discuss developer-first infrastructure and why it matters"
+date: 2022-04-13T17:46:27-05:00
+meta_desc: |
+    Explore how James Governor, Redmonk, and Joe Duffy, Pulumi, think about
+    developer-first infrastructure.
+meta_image: meta.png
+authors:
+    - george-huang
+tags:
```
Comment (gthuang) [inline themes/default/content/blog/redmonk-pulumi-developer-first-infra/index.md:10]: can we add these tags?
cloud-computing
infrastructure as code

File: themes/default/content/blog/redmonk-pulumi-developer-first-infra/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Pulumi and RedMonk discuss developer-first infrastructure and why it matters"
+date: 2022-04-13T17:46:27-05:00
+meta_desc: |
```
Comment (gthuang) [inline themes/default/content/blog/redmonk-pulumi-developer-first-infra/index.md:4]: update to: 

Pulumi and RedMonk discuss how developer-first infrastructure means empowering developers to build and deploy applications through software engineering practices in cloud computing.

File: themes/default/content/blog/redmonk-pulumi-developer-first-infra/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+1. Secure and Observable By-Construction
+1. An API for Everything
+
+[image]
+
+Finally, Joe dives into the best practices of developer-first infrastructure:
+
+1. **Unified Engineering:** Software engineering tools and practices for applications and infrastructure.
+1. **APIs:** Making everything programmable and composable with APIs.
+1. **Everything as Code:** Use "as code" techniques like infrastructure and policy as code.
```
Comment (gthuang) [inline themes/default/content/blog/redmonk-pulumi-developer-first-infra/index.md:48]:
```suggestion
1. **Everything as Code:** Use "as code" techniques like infrastructure as code and policy as code.
```

**Comment by cnunciato** [comment]
Didn't realize this was here @nimbinatus! I've added you as a co-author to #1420, so I'll close this PR.

**Pull request closed without merging**